screens/HealthProfessionalScreen.js

```diff
@@ -1,158 +1,131 @@
-import React, { useState } from 'react';
-import { View, Text, TextInput, TouchableOpacity, Dimensions, StyleSheet, Keyboard } from 'react-native';
-
+import React, { useState }from 'react';
+import { View, Text, TextInput, TouchableOpacity, Image, Button, Dimensions, StyleSheet, Keyboard } from 'react-native';
+import MenuIcon from '../assets/images/menu-icon';
 import * as Animatable from 'react-native-animatable';
-import { firebase } from '../components/Firebase/config';
-
-import Background from '../components/hpBackground';
-import NurseEllieLogo from '../assets/images/nurse-ellie-logo.svg';
-
-import MenuIcon from '../assets/images/hp-menu-icon.svg';
-import BlueAddIcon from '../assets/images/blue-add-icon';
+import { firebase } from '../components/Firebase/config'
+import Background from '../components/background';
```

Comment:
Use hpBackground instead. Since this is the health professional screen not patient screen. 

```suggestion
import Background from '../components/hpBackground';
```